octavia-cli/octavia_cli/apply/resources.py

```diff
@@ -427,8 +447,85 @@ def update_payload(self) -> DestinationUpdate:
+            dict: Remote destination configuration.
+        """
+        comparable_configuration = super()._get_comparable_configuration()
+        return comparable_configuration.connection_configuration
+
+
+class Connection(BaseResource):
+    CHECK_RETURN_TYPE = False
+    APPLY_PRIORITY = 1
+    api = connection_api.ConnectionApi
+    create_function_name = "create_connection"
+    resource_id_field = "connection_id"
+    search_function_name = "search_connections"
+    update_function_name = "update_connection"
+    resource_type = "connection"
+
+    @property
+    def status(self) -> ConnectionStatus:
+        return ConnectionStatus(self.local_configuration["configuration"]["status"])
+
+    @property
+    def create_payload(self) -> ConnectionCreate:
+        """Defines the payload to create the remote connection.
+        Disable snake case parameter usage with _spec_property_naming=True
+
+        Returns:
+            ConnectionCreate: The ConnectionCreate model instance
+        """
+        return ConnectionCreate(**self.configuration, _check_type=False, _spec_property_naming=True)
+
+    @property
+    def search_payload(self) -> ConnectionSearch:
+        """Defines the payload to search the remote connection. Search by connection name if no state found, otherwise search by connection id found in the state.
+        Returns:
+            ConnectionSearch: The ConnectionSearch model instance
+        """
+        if self.state is None:
+            return ConnectionSearch(
+                source_id=self.source_id, destination_id=self.destination_id, name=self.resource_name, status=self.status
+            )
+        else:
+            return ConnectionSearch(connection_id=self.state.resource_id, source_id=self.source_id, destination_id=self.destination_id)
+
+    @property
+    def update_payload(self) -> ConnectionUpdate:
+        """Defines the payload to update a remote connection.
+
+        Returns:
+            ConnectionUpdate: The DestinationUpdate model instance.
+        """
+        return ConnectionUpdate(
+            connection_id=self.resource_id,
+            sync_catalog=self.configuration["syncCatalog"],
+            status=self.configuration["status"],
+            namespace_definition=self.configuration["namespaceDefinition"],
+            namespace_format=self.configuration["namespaceFormat"],
+            prefix=self.configuration["prefix"],
+            schedule=self.configuration["schedule"],
+            resource_requirements=self.configuration["resourceRequirements"],
+            _check_type=False,
+        )
+
+    def _get_comparable_configuration(self):
+        """Get the object to which local configuration will be compared to.
+
+        Returns:
+            dict: Remote connection configuration.
+        """
+        comparable_configuration = super()._get_comparable_configuration()
+        comparable_configuration.pop("connectionId")
+        comparable_configuration.pop("operationIds")
```

Comment:
Today octavia-cli can't work with normalization right?

Reply:
Exactly, I remove the `operationIds` in case the user change the transformation in the UI, so we exclude operationIds from the diff computation.